css/project-page.styl

```diff
@@ -1,3 +1,22 @@
+.project-about-page
+  background: rgba(255,255,255,0.95)
+  border-radius: 3px
+  color: #000000
+  margin: 1.5em 12vw
+  padding: 1.5em 6vw
+
+  .markdown
+    img
+      max-width: 100%
+
+    h1:first-child, h2:first-child,
```

Comment:
Given the design, I think you want to use the `:first-of-type` pseudo-class rather than `:first-child`